tests/config/loader/ini/replace/test_replace_posargs.py

```diff
@@ -5,18 +5,27 @@
 from tests.config.loader.ini.replace.conftest import ReplaceOne
 
 
-def test_replace_pos_args_none_sys_argv(replace_one: ReplaceOne) -> None:
-    result = replace_one("{posargs}", None)
+@pytest.mark.parametrize(
+    "syntax", ["{posargs}", pytest.param("[]", marks=pytest.mark.xfail(raises=AssertionError))]  # noqa: SC200
```

Comment:
don't silence error instead add to whitelist https://github.com/tox-dev/tox/blob/rewrite/whitelist.txt

Reply:
I'm not a fan of all this noise on the PRs. Can you just make a final commit when you're close to done?

Reply:
Development should happen locally, the CI should only be used to do the final validation (let's not overload Github system needlessly).